ci_framework/roles/operator_build/tasks/build.yml

```diff
@@ -157,10 +157,6 @@
     params:
       IMG: "{{ operator_img }}"
       BUNDLE_IMG: "{{ operator_img_bundle }}"
-      IMAGENAMESPACE: "{{ cifmw_operator_build_push_org }}"
```

Comment:
we need those image var for bundle and catalog_build both targets only and not bundle_build as openstack-operator bundle does not include other csvs now